recipe/meta.yaml

```diff
@@ -1,61 +1,57 @@
-{% set version = "3.7.1" %}
+{% set name = "twine" %}
+{% set version = "4.0.2" %}
 
 package:
-  name: twine
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/t/twine/twine-{{ version }}.tar.gz
-  sha256: 28460a3db6b4532bde6a5db6755cf2dce6c5020bada8a641bb2c5c7a9b1f35b8
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/twine-{{ version }}.tar.gz
+  sha256: 9e102ef5fdd5a20661eb88fad46338806c3bd32cf1db729603fe3697b1bc83c8
 
 build:
-  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
-  skip: true  # [py<36]
```

Comment:
We might want to keep this for documentation purposes. Though I agree it's not necessary as we aren't building for the lowest version of python 3.7.

Reply:
ty, addressed